### DISCUSSION TITLE
What's the long-term future for EME?

### DISCUSSION THREAD

**Discussion by dwiig** [liedekef/events-made-easy#442] [General]
We've used Events Made Easy for years on three sites, love the plug-in and it fits our needs very well. (We've donated.) We're wondering if EME will continue to be supported and updated for the long term, considering the WordPress problems. We're redesigning one website and launching another and if we're going to have to replace EME with another plug-in at some point in the near future, this would be the time to make the switch. Franky, should we count on EME being supported for a long time, or should we look for an alternative?  Thanks.

**Comment by liedekef** [comment]
This is a weird question as nobody can foresee the future. I don't intend on stopping any time soon, but your question can be asked to just about any plugin developer and none will be able to give you a 100% guarantee on the future :-)

**Reply by dwiig** [reply]
Thanks for your reply. Certainly can't expect a guarantee, but I'm going to rely on your response to plan to stick with EME on our upcoming investment.  Thanks.

**Reply by liedekef** [reply]
As a side note: EME exists for over 10 years already (I think I started in 2011 or so, forking the at-that-time dead Events Manager), and it has become something very good if I say so myself (and still 100% free, even with all the payment gateways included, membership, custom formfields etc ... most other plugins charge you for all that). But I don't claim to be the best though, just the best free can buy :-D

**Reply by dwiig** [reply]
We agree!
